src/calibre/utils/spell/hunspell_wrapper.cpp

```diff
@@ -123,65 +129,48 @@ static PyMethodDef HunSpell_methods[] = {
 };
 
 static PyTypeObject DictionaryType = {
-	PyObject_HEAD_INIT(NULL)
```

Comment:
... turns out it's because this is C++ code, and compilers didn't start supporting this until fairly recently. Neither MSVC 2015 or gcc 4.8 can use this syntax. Will revert.